_pkgdown.yml

```diff
@@ -356,3 +356,19 @@ reference:
   - get_window_end_commit
   - get_window_size
   - get_window_start_commit
+- title: __OpenHub__
+  desc: >
+   Functions to interact and download data from OpenHub's API, Ohloh.
+   See [OpenHub Project Search](../articles/openhub_project_search.html)
+   Notebook for an example.
+- contents:
+  - openhub_api_organizations
+  - openhub_api_portfolio_projects
+  - openhub_api_projects
+  - openhub_api_analyses
```

Comment:
can you confirm all the api_ functions save the raw, unmodified file to disk? I am assuming they are XMLs, or was it JSONs? It is important the downloaders do not do any pre-processing to the data.

Reply:
The response files are all XML files and all of the openhub_api_* functions are downloaders that do not do any pre-processing to the data.